packages/legend-graph/src/graph/PureModel.ts

```diff
@@ -581,6 +587,34 @@ export class PureModel extends BasicModel {
     return element;
   }
 
+  getElementOrigin(
```

Comment:
this and the one in dependency manager's method signature make little sense, let's just do `getElementOrigin(element: PackageableElement): string | undefined`